From ff476abd0abeb1d100381b7981aaf5cbf7a40a6b Mon Sep 17 00:00:00 2001
From: Mingliang Liu <liuml07@apache.org>
Date: Tue, 20 Sep 2016 13:19:44 -0700
Subject: [PATCH 2048/2748] HADOOP-13601. Fix a log message typo in
 AbstractDelegationTokenSecretManager. Contributed
 by Mehran Hassani.

(cherry picked from commit e80386d69d5fb6a08aa3366e42d2518747af569f)
(cherry picked from commit 2a1e48bd2f71066bbba5708825c5f6a09cae62a7)

Change-Id: I3e35931cc11c713d395ffa5326509dacd6c1b8e6
---
 .../AbstractDelegationTokenSecretManager.java      |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/token/delegation/AbstractDelegationTokenSecretManager.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/token/delegation/AbstractDelegationTokenSecretManager.java
index ac399ec..88aebd4 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/token/delegation/AbstractDelegationTokenSecretManager.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/token/delegation/AbstractDelegationTokenSecretManager.java
@@ -520,7 +520,7 @@ public synchronized TokenIdent cancelToken(Token<TokenIdent> token,
     DataInputStream in = new DataInputStream(buf);
     TokenIdent id = createIdentifier();
     id.readFields(in);
-    LOG.info("Token cancelation requested for identifier: "+id);
+    LOG.info("Token cancellation requested for identifier: " + id);
     
     if (id.getUser() == null) {
       throw new InvalidToken("Token with no owner");
-- 
1.7.9.5

